sdk/appplatform/Microsoft.Azure.Management.AppPlatform/src/Microsoft.Azure.Management.AppPlatform.csproj

```diff
@@ -6,28 +6,33 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.AppPlatform</PackageId>
     <Description>Provides developers with libraries for the Azure Spring Cloud under Azure Resource manager to deploy, start, stop and scale Spring Cloud applications.</Description>
-    <Version>1.0.0</Version>
+    <Version>1.1.0-preview</Version>
     <AssemblyName>Microsoft.Azure.Management.AppPlatform</AssemblyName>
     <PackageTags>management;spring;spring cloud;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        Stable release for Azure Spring Cloud SDK.
-
-        1) Move ConfigServerProperties from ClusterResourceProperties as a separate resource ConfigServerResource.
-
-        2) Move TraceProperties from ClusterResourceProperties as a separate resource MonitoringSettingsResource.
-
-        3) Add support for VNet injected.
-
-        4) Add support for CustomDomain.
-
-        5) Add support for ManagedIdentity.
+        Preview release for Azure Spring Cloud SDK.
       ]]>
     </PackageReleaseNotes>
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
   </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
```

Comment:
Why the test framework project is included in the source?

Reply:
deleted them.